clients/js/src/generated/instructions/createV1.ts

```diff
@@ -121,13 +121,13 @@ export type CreateV1InstructionDataArgs = {
   creators: OptionOrNullable<Array<CreatorArgs>>;
   primarySaleHappened?: boolean;
   isMutable?: boolean;
-  tokenStandard: TokenStandardArgs;
+  tokenStandard?: TokenStandardArgs;
```

Comment:
Is there a way to preserve the function signature for createV1?

Reply:
Not sure. It's changed in the reference PR. So, I thought it would be OK. I'll check that.